hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleToken.java

```diff
@@ -718,6 +726,200 @@ public List<CustomFee> grpcFeeSchedule() {
         return grpcList;
     }
 
+    public EvmTokenInfo asEvmTokenInfo(final ByteString ledgerId) {
```

Comment:
These helper methods should be moved to a helper class and should not be in `MerkleToken`